### PR TITLE
docs: added meeting minutes office hour 21.06.2024

### PR DESCRIPTION
**Pull request by matbmoser**
### Added
- Office hour meeting minutes from 21.06.2024

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: blog-meeting-minutes/2024-06-21-community-hour.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- 1.000 Lines of code in PRs for non-contributors require project Ip check: https://www.eclipse.org/projects/handbook/#ip-project-content
+
+### Open planning / community
+
+- Every office hour there will be a slot to talk about the current process of the working model of Tractus-X/Catena-X. With the updates from the community.
+
+### Discussions
+
+- Mermaid version in current docussaurus for the tractus-x webpage do not support specific type block-beta and xychart-beta.
+  - [Stephan Bauer](https://github.com/stephanbcbauer) will test to upgrate the version of docussaurus in the Catena-X eV repo
```
Comment (evegufy) [inline blog-meeting-minutes/2024-06-21-community-hour.md:37]: typo upgrade

File: blog-meeting-minutes/2024-06-21-community-hour.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- No security team will be available for R24.08, [Rohan Krish](https://github.com/RoKrish14) has told he will be going in July, contact him if you know who will be responsible for security in the Association.
+- Handover from [Rohan Krish](https://github.com/RoKrish14) will be done in this ticket: https://github.com/eclipse-tractusx/sig-security/issues/83
+
+### FOSS
+
+- Committer Election for [Lucas Capellino](https://github.com/arnoweiss), please vote if you are a committer!
+- [Evelyn Gurschler](https://github.com/evegufy) election to project lead concluded successfully, congratulations to our new project lead!
+- [Committer Election](https://projects.eclipse.org/projects/automotive.tractusx/elections/election-lucas-capellino-eclipse-tractus-x) for [Lucas Capellino](https://github.com/ds-lcapellino) open for voting again
+- Don't forget to update the legal docs regarding CC-BY-4.0 for documentation! Close the tickets in your repositories if its done: [eclipse-tractusx/sig-infra#477](https://github.com/eclipse-tractusx/sig-infra/issues/477)
```
Comment (evegufy) [inline blog-meeting-minutes/2024-06-21-community-hour.md:27]: the change is mainly regarding CC-BY-4.0 but not only, I'd shorten to "Don't forget to update the legal docs!"

**Review by evegufy — CHANGES_REQUESTED** [review]
LGTM in general just some small comments

File: blog-meeting-minutes/2024-06-21-community-hour.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+### Infrastructure
+
+- Info from Test Management will be taken over by [Harald Zimmer](https://github.com/ds-hzimmer), please contact with your product owners to get a meeting with Harald and get the access if you are in the association.
+- Feedback requested from [Harald Zimmer](https://github.com/ds-hzimmer) for who is using: https://github.com/catenax-ng/product-test-data-generator
+- Branch protecting example provided by [Tom Meyer](https://github.com/tom-rm-meyer-ISST)
+
+### Security team
+
+- No security team will be available for R24.08, [Rohan Krish](https://github.com/RoKrish14) has told he will be going in July, contact him if you know who will be responsible for security in the Association.
+- Handover from [Rohan Krish](https://github.com/RoKrish14) will be done in this ticket: https://github.com/eclipse-tractusx/sig-security/issues/83
```
[the remark this answers is not in the document]
Comment (matbmoser) [reply blog-meeting-minutes/2024-06-21-community-hour.md:20]: ok

File: blog-meeting-minutes/2024-06-21-community-hour.md (hunk 2 above)
Comment (matbmoser) [reply blog-meeting-minutes/2024-06-21-community-hour.md:27]: ok

File: blog-meeting-minutes/2024-06-21-community-hour.md (hunk 1 above)
Comment (matbmoser) [reply blog-meeting-minutes/2024-06-21-community-hour.md:37]: done

**Comment by matbmoser** [comment]
Thanks for the review @RoKrish14 and @evegufy :)

**Review by RoKrish14 — APPROVED** [review]
LGTM

**Review by evegufy — APPROVED** [review]

**Pull request merged**